src/Publisher/FilesystemPublisher.php

```diff
@@ -68,7 +68,7 @@ public function purgeURL($url)
     {
         if (!$url) {
             user_error('Bad url:' . var_export($url, true), E_USER_WARNING);
-            return;
+            return [];
```

Comment:
Why is this change here? It seems out of scope for adding extension points. If there is a valid reason to change what is returned here, please create a separate PR for this purpose which can be reviewed and merged on its own merits.
```suggestion
```